src/clientversion.cpp

```diff
@@ -74,6 +74,7 @@ std::string FormatSubVersion(const std::string& name, int nClientVersion, const
             return "Knots:" + CLIENT_BUILD.substr(pos + 6) + "/";
         }();
         ua += ua_knots;
+        ua += "UASF-ReducedData:0.1/";
```

Comment:
This makes sense if you're going to have an activation client, but not so much for a merge into mainline Knots.